Realm/Realm.Sync/SyncConfiguration.cs

```diff
@@ -87,11 +109,18 @@ internal override Realm CreateRealm(RealmSchema schema)
 
         private Native.SyncConfiguration ToNative()
         {
+            if (!string.IsNullOrEmpty(TrustedCAPath) &&
+                !File.Exists(TrustedCAPath))
+            {
+                throw new RealmFileNotFoundException($"File not found: {TrustedCAPath}");
```

Comment:
Right. Well, I can't think of any reason not to use the system one, unless we for some reason want all exceptions thrown by our SDK to be `Realm*`-exceptions. But that just seems silly to me. As does prefixing them with `Realm` in general, unless they are related to something in a realm, not just in the SDK.